homeassistant/components/tradfri/cover.py

```diff
@@ -29,6 +30,14 @@ def __init__(self, device, api, gateway_id):
 
         self._refresh(device)
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attr = {}
+        attr[ATTR_BATTERY_LEVEL] = self._device_info.battery_level
```

Comment:
Please remove battery level.

Reply:
@MartinHjelmare .. I did ask you about making a new entity for the battery here:
https://github.com/home-assistant/core/pull/33674#discussion_r403647130
Thanks

Reply:
Our current design guidelines is to not include battery level as an attribute but as separate sensors. We will not have a discussion about this in this PR. You're welcome to make an issue in our architecture repo if you have a suggestion to change the guidelines and want to invest time to implement that change.

Reply:
That isn't what I was referring to:

> Sorry if I came across a little strong ... but I have just been looking through the 2 battery entities that I have that are separate... and just noticed that the [Home Assistant Mobile App Support](https://www.home-assistant.io/integrations/mobile_app/) integration has the battery as an attribute as well as a separate entity; I am not opposed to doing this as it would allow people to query the battery using the method of their choice.. How can I add the battery as an entity to the cover device (I have not done anything with devices before and am fairly new to programming HA integrations)?
Thanks, Kris

Reply:
Sorry, I misunderstood. I've replied here:
https://github.com/home-assistant/core/pull/33674#discussion_r403764538

Reply:
If we remove this I think we can merge this PR.

Reply:
Still have to add the battery as a seperate entity.. been flat out with work the last couple of days and completely drained.. so will do it over the Easter holiday 🙂

Reply:
That should be a separate PR. It should not go into this PR.

Reply:
Ok will do separately in another PR ...
To match this I have updated the title since the battery will now be moved out!